### DISCUSSION TITLE
Use Pest for tests

### DISCUSSION THREAD

**Discussion by ismaail** [shyamsitaula/samarium#118] [Ideas]
Are you interested in swiching to **Pest** for Unit Tests (to replacel PHPUnit) ?

or stick with PHPUnit ?

**Comment by shyamsitaula** [comment]
Sorry for late response. Missed the discussion tab somehow. 

I have no problem with PHPUnit so if an option I would stick with it. If switching to Pest is really worthy then I am open to it too. Honestly, need to add some test codes so whatever aids in that process I would be glad.

Thanks.

**Comment by rasheedahmad7** [comment]
can I write a pest test for this project

**Comment by shyamsitaula** [comment]
@rasheedahmadmlik Thanks for your interest in this project. 

I would prefer sticking with PHPUnit for now. Could you add more test code with PHPUnit? I guess PHPUnit tests can be updated later to use Pest? 

Your thoughts?

**Comment by ismaail** [comment]
For now we need to fix failing tests.

**Comment by ismaail** [comment]
check this PR #125
